### PR TITLE
Removing Program.cs from Unit Test template for .NET core.

### PR DESCRIPTION
**Pull request by singhsarab**
1. Removing the Program.cs from the Unit Test template for .NET core for CSharp.
2. Updating the reference to Microsoft.NET.Test.Sdk package which brings in the targets for auto-generating the Program.cs.

### CODE REVIEW

**Comment by singhsarab** [comment]
DONOT Merge until @codito has signed off. Thanks :)

**Comment by eerhardt** [comment]
What's the testing process for these templates?  Is it completely a manual test to ensure nothing is broken?

**Comment by singhsarab** [comment]
@eerhardt, We already have some smoke tests as part of our nightly runs that run daily. For new scenarios, we are adding more tests and will continue to do so.

**Review by srivatsn — APPROVED** [review]

**Review by eerhardt — APPROVED** [review]

**Comment by srivatsn** [comment]
Waiting for @codito to approve before merging per @singhsarab's comment.

**Pull request merged**
